StPetersburg/InEtn56274.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"
+?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0"
+     xml:lang="en"
+     xml:id="InEtn56274"
+     type="mss">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Medical Instructions</title>
+            <editor key="DE"/>
+            <editor role="generalEditor" key="AB"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed
+            under the Creative Commons Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0524InEtn"/>
+                  <idno>InEtn 5627-4</idno>
+                  <altIdentifier>
+                     <idno>Platonov cat. III.111</idno>
+                  </altIdentifier>
+               </msIdentifier>
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                    <locus from="2r" to="12r"/>
+                     <title type="complete" ref="LIT5842MedicalAdvice"></title>
+                     <textLang mainLang="am"/>
+                     <colophon xml:id="coloph1" xml:lang="am">
+                       <locus from="2v" to="3r"></locus>
+                       ከመስኮብ፡ ቋንቋ፡ በሀበሻ፡ ቋንቋ፡ የተረጐመው፡ <persName ref="PRS12879Bazzabbeh" role="translator">በዛብህ፡</persName> ነው።
+                       <date calendar="ethiopian" when-custom="04-14">በታኅሣሥ፡ በ፲፬ ቀን፡</date>
```

Comment:
I am not sure how this date should be marked up. The calendar "ethiopian" refers to the ʿĀmata śəggāwe, and here there is no reference to the year, only the month and day, so I think this value should better not be used. There would be the possibility for the mark up with ethiocal:Tahsas14, which has been used until now for recurring dates for liturgical readings etc (not a precise, but incomplete date as here).

Reply:
corrected, i used the format provided by TEI for such incomplete dates.